### PR TITLE
BETA: Register smartcloud.is-a.dev

### PR DESCRIPTION
**Pull request by Smartlinuxcoder**
Added `smartcloud.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: package-lock.json
Comment (DEV-DIBSTER) [inline package-lock.json:1]: This file should not be touched.

File: .vscode/settings.json
Comment (DEV-DIBSTER) [inline .vscode/settings.json:1]: This file should not be touched.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
Comment (Smartlinuxcoder) [reply .vscode/settings.json:1]: Ah, I see, my vscode created that file

File: package-lock.json
Comment (DEV-DIBSTER) [reply package-lock.json:1]: This file still needs to be restored.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Pull request closed without merging**